### PR TITLE
Add S3 bucket support.

### PR DESCRIPTION
**Pull request by ildyria**

### CODE REVIEW

File: config/secure-headers.php [hunk 1]
```diff
@@ -377,19 +379,27 @@
...
+					'https://b.osm.rrze.fau.de/osmhd/',
+					'https://c.osm.rrze.fau.de/osmhd/',
+					'data:', // required by openstreetmap
+					'blob:', // required for "live" photos
+				],
+				// Add the S3 URL to the list of allowed image sources
+				env('AWS_ACCESS_KEY_ID', '') === '' ? [] :
+				[
+					// @phpstan-ignore-next-line
+					str_replace(parse_url(env('AWS_URL'), PHP_URL_PATH), '', env('AWS_URL')),
```
Comment (d7415) [inline config/secure-headers.php:400]: ~~Does this not remove the URL? Other than maybe `:///`~~

No, ignore me.

Comment (d7415) [reply config/secure-headers.php:400]: Nevermind.

**Review by d7415 — APPROVED** [review]

**Comment by Kovah** [comment]
@ildyria It's sad to be not even mentioned a bit for the early contributions and testing of the feature.  I spent days working on this. After this experience, I will never contribute again to Lychee in the future.

ed2

**Comment by d7415** [comment]
@Kovah I think that was an oversight (possibly automated). The [release notes on the website](https://github.com/LycheeOrg/LycheeOrg.github.io/pull/126/files) were written (by ildyria) to credit you.

**Comment by ildyria** [comment]
@Kovah **I literally made sure that you were in the git history while doing the squash merge.**
image
and also added you (before you complained) in the release as above mentioned by @d7415 (see time stamp of the commit in the branch).

The release notes of the github are automated. 
I will fix it.

**Pull request merged**